samples/COVID/covid-spfx/src/webparts/covidAdmin/components/organisms/ContactTracing.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import { Logger, LogLevel } from "@pnp/logging";
+
+import cloneDeep from "lodash/cloneDeep";
+import isEqual from "lodash/isEqual";
+import find from "lodash/find";
+import forEach from "lodash/forEach";
+import { Dictionary } from "lodash/index";
+
+import strings from "CovidWebPartStrings";
+import { cs } from "../../services/covid.service";
+import { IQuery, ICheckIns } from "../../models/covid.model";
+import TableHeader from "../atoms/TableHeader";
+import TableSectionHeader from "../atoms/TableSectionHeader";
+import TableSection from "../atoms/TableSection";
+import Search from "../molecules/Search";
+import { IDropDownOption } from "../atoms/DropDown";
+
+export interface IContactTracingProps {
+
+}
+
+export interface IContactTracingState {
+  allExpanded: boolean;
+  searchResults: Dictionary<ICheckIns[]>;
```

Comment:
for example, replace this with
searchResults: { [key: string]: ICheckIns[]}

That will save you 25kb of js in your ACE / webpart.  With 10-20 cards, that can save a bunch of bytes and processing.